gneiss/regression/_transformer.py

```diff
@@ -0,0 +1,33 @@
+from ._ols import OLSModel
+from ._mixedlm import LMEModel
+from ._format import RegressionFormat_g
+from gneiss.plugin_setup import plugin
+
+
+@plugin.register_transformer
+def _1(data: OLSModel) -> RegressionFormat_g:
+    ff = RegressionFormat_g()
+    with ff.open() as fh:
+        data.write_pickle(fh)
+    return ff
```

Comment:
Not sure what is going on - but it seems to appear in quite a few projects

https://github.com/scikit-learn/scikit-learn/issues/3410
https://github.com/python-pillow/Pillow/issues/477
https://github.com/Supervisor/supervisor/issues/391